src/actions.rs

```diff
@@ -16,6 +17,10 @@ use setup::{get_setup_menu, setup_inquire_and_handle};
 use strum::IntoEnumIterator;
 use strum_macros::{Display, EnumIter};
 use transaction::TransactionActions;
+use alloy::primitives::Address;
+use crate::actions::send_message::send_message;
+use crate::network::Network; // Assuming `Network` is defined in a `network` module
```

Comment:
Please remove any unhelpful comments generated by AI like this one

```suggestion
use crate::network::Network;
```